src/bucket/BucketIndex.cpp

```diff
@@ -107,7 +108,7 @@ BucketIndexImpl<IndexT>::BucketIndexImpl(BucketManager const& bm,
         bloom_parameters params;
         params.projected_element_count = estimatedNumElems;
         params.false_positive_probability = 0.001; // 1 in 1000
-        crypto_shorthash_keygen(params.random_seed.data());
+        crypto_shorthash_keygen(shortHash::getShortHashInitKey().data());
```

Comment:
this is not what you want, this should be `params.random_seed = shortHash::getShortHashInitKey()`